nuxt-frontend/components/backend-development/PartnerWithUsSection.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <section class="bg-[#121212] 3xl:outer-container">
+    <div class="py-14 lg:py-[7.5rem] container text-white">
+      <!-- ---------------------MOBILE UI START------------------ -->
+      <div class="flex flex-col gap-8 lg:hidden">
+        <h2 class="mobile-header-1 lg:desk-header-1 text-center">
+          Why should you partner with Canopas?
+        </h2>
+        <div class="flex flex-col gap-6" v-for="item in items" :key="item">
+          <div class="flex gap-4">
+            <img
+              :src="item.image"
+              class="w-9 h-9 object-contain"
+              :alt="item.title"
+            />
+            <div class="flex flex-col gap-3">
+              <span class="sub-h1-semibold lg:mobile-header-2-semibold">{{
+                item.title
+              }}</span>
+              <span
+                class="sub-h3-regular text-white-core-80"
+                v-html="item.description"
+              ></span>
+            </div>
+          </div>
+        </div>
+        <nuxt-link
+          class="gradient-btn primary-btn mt-0"
+          to="/contact"
+          @click.native="$mixpanel.track('tap_backend_partner_with_us_cta')"
+        >
+          <span class="sub-h3-semibold">Let’s Work Together</span></nuxt-link
+        >
+      </div>
+      <!-- ---------------------MOBILE UI END-------------------- -->
+
+      <!-- ---------------------DESKTOP UI START------------------ -->
+      <div id="partner-with-us" class="hidden lg:block h-[150vh]">
+        <div class="sticky top-0 h-[110vh] xl:top-4 xl:h-[100vh] main">
+          <div
+            class="absolute -translate-y-1/2 top-1/2"
+            :class="[
+              isAnimated
+                ? '!flex flex-1 !text-white'
+                : 'left-1/2 -translate-x-1/2',
+            ]"
+          >
+            <div class="flex-col my-auto" :class="{ '!flex': isAnimated }">
+              <span
+                class="header-text font-normal xl:!font-bold text-white/[0.12]"
+                :class="{
+                  '!text-white': isAnimated,
+                }"
+                >Why should you</span
+              >
+              <span
+                class="block ml-2 transition-all duration-1000 text-white/[0.12]"
+                :class="[
+                  isAnimated
+                    ? '!text-white !header-text font-normal xl:!font-bold !ml-0'
+                    : 'text1',
+                ]"
+                >partner
+              </span>
+              <div
+                class="flex ml-0 xl:ml-32 gap-6"
+                :class="{ '!ml-0 block': isAnimated }"
+              >
+                <span
+                  class="my-auto -rotate-90 header-text font-normal xl:!font-bold text-white/[0.12]"
```

Comment:
if white or black color is having any new opacity add it in tailwind config file and use it from there